nemo_rl/models/policy/dtensor_policy_worker.py

```diff
@@ -640,14 +672,21 @@ def train(
 
                     with DTensorPolicyWorker.train_context(context_parallel_ctx):
                         with torch.autocast(device_type="cuda", dtype=self.dtype):
-                            outputs = self.model(
-                                input_ids=input_ids,
-                                attention_mask=attention_mask,
-                                position_ids=position_ids,
-                                use_cache=False,
-                                flash_attn_kwargs=flash_attn_kwargs,
-                            )
-
+                            if self.cfg["reward_model_type"] == "bradley_terry":
```

Comment:
would sequence packing work with this? I suspect no due to the flash_attn_kwargs. Can we guard it?